ray_on_golem/server/main.py

```diff
@@ -158,6 +164,24 @@ async def ray_service_ctx(app: web.Application) -> None:
     await ray_service.shutdown()
 
 
+async def reputation_service_ctx(app: web.Application) -> None:
+    reputation_service: ReputationService = app["reputation_service"]
+
+    await reputation_service.start()
+    updaters = [
+        ReputationUpdater(network) for network in ["polygon", "mumbai", "goerli", "holesky"]
+    ]
+    update_tasks = [asyncio.create_task(updater.update()) for updater in updaters]
```

Comment:
Use `create_task_with_logging` from golem-core and also give it a proper `trace_id`.